### PR TITLE
Display media runtime in hour minute format

### PR DESCRIPTION
**Pull request by dtrexler**
# Display media runtime in hour minute format

**Changes**
Updates the runtime display in the FullDetailsFragment to a ```[0-9]+h [0-9][0-9]?m``` format in order to be consistent with the web client.

**Issues**
Fixes issue https://github.com/jellyfin/jellyfin-androidtv/issues/2806
Supersedes pull request https://github.com/jellyfin/jellyfin-androidtv/pull/3878

### CODE REVIEW

File: app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/FullDetailsFragment.java [hunk 1]
```diff
@@ -678,7 +678,25 @@ public void setTitle(String title) {
...
+
+        int totalMinutes = (int) Math.ceil((double) runtime / 600000000);
+
+        int hours = totalMinutes / 60;
+        int minutes = totalMinutes % 60;
+
+        String runtimeString = "";
+
+        if (hours > 0) {
+            runtimeString += hours + getString(R.string.hours_shorthand) + " ";
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/FullDetailsFragment.java:694]: Use string formatting instead so translators can place the digit behind the shorthand if their language needs that. Something like `%1$dh %1$dm` for the hour+minute string should work.

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -64,7 +64,8 @@
     <string name="lbl_watched">Watched</string>
     <string name="lbl_sd">SD</string>
     <string name="lbl_runs">"Runs"</string>
-    <string name="lbl_min">" min"</string>
+    <string name="minutes_shorthand">"m"</string>
+    <string name="hours_shorthand">"h"</string>
```
[the remark this answers is not in the document]
Comment (dtrexler) [reply app/src/main/res/values/strings.xml]: Done. New strings do not have quotes

File: app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/FullDetailsFragment.java (hunk 1 above)
Comment (dtrexler) [reply app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/FullDetailsFragment.java]: Ah, I see. Thanks for the feedback. Changes completed as requested.

**Review by nielsvanvelzen — APPROVED** [review]
Thanks for the PR! I'll merge it once development for 0.19.z starts (somewhere in the coming days after 0.18.3 is released). No need to do any merges with master.

**Pull request merged**